elasticgraph-graphql/lib/elastic_graph/graphql/http_endpoint.rb

```diff
@@ -196,8 +196,8 @@ def normalized_headers
         end
       end
 
-      def content_type
-        normalized_headers["CONTENT-TYPE"]
+      def mime_type
```

Comment:
You'll need to update https://github.com/block/elasticgraph/blob/d33b56221662cf54730c1179288aa6845e52c52a/elasticgraph-graphql/sig/elastic_graph/graphql/http_endpoint.rbs#L87 to `mime_type` to match.

Reply:
thanks, done